artifacthub-repo.yaml

```
@@ -2,10 +2,10 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-only
 
-repositoryID: f900b890-eb67-4328-9100-336301f7cefe
+repositoryID: 6a910e71-6940-45a6-9d63-07ec0080aa55
 owners:
   - name: Element Server Products Team
     email: onprem-support@element.io
 ignore:
   - name: matrix-stack
-    version: "[0-9]+\\.[0-9]+\\.[0-9]+\\..+"
+    version: "[0-9]+\\.[0-9]+\\.[0-9]+-sha.+"
```

```suggestion
    version: "[0-9]+\\.[0-9]+\\.[0-9]+-(sha.+|dev)"
```

We've got to hide the legacy `-dev` artifacts too yes?